build.gradle

```diff
@@ -239,7 +242,7 @@ pitest {
     historyOutputLocation = 'build/reports/pitest/fastermutationtestingoutput'
     outputFormats = ['XML', 'HTML']
     timestampedReports = false
-    mutationThreshold = 85
+    mutationThreshold = 80
```

Comment:
Have you created a ticket to improve the coverage later? Hope @kavithagade12 @pksaxena01 @shreedhar-hmcts are aware of this

Reply:
yes , will take care of this .